integration-tests/manager-integration-tests/manager-integration-tests-v2/README.md

```diff
@@ -0,0 +1,141 @@
+# E2E Integration tests for API v2
+
+Module contains integration tests verifying end to end functionality in real cluster.
+
+## Test design
+
+Tests are written using [Gherkin syntax](https://cucumber.io/docs/gherkin/reference/).
+Test steps are implemented in Java using [Cucumber JVM framework](https://github.com/cucumber/cucumber-jvm).
+For connection to external systems the test suite leverages [TNB - System X](https://github.com/tnb-software/TNB/tree/main/system-x).
+
+### Test scenarios
+
+Main test scenarios are stored in [features folder](src/test/resources/features).
+
+Scenarios are split into dedicated feature files:
+- [bridge.feature](src/test/resources/features/bridge.feature) - Contains CRUD tests for Bridge
+- [ingress.feature](src/test/resources/features/ingress.feature) - Contains basic tests for exposed Bridge endpoint
+- [processor.feature](src/test/resources/features/processor.feature) - Contains CRUD tests for Processor
+
+Rest of scenarios are kept in two different feature folders:
+
+- [performance-features](src/test/resources/performance-features) - Contains different performance test scenarios
+- TO-DO describe the rest after the implementation evolves
+
+### Test steps implementation
+
+Main tests are executed as [JUnit 5 test suite](src/test/java/com/redhat/service/smartevents/integration/tests/RunCucumberTest.java).
+
+Test steps are stored in [steps folder](src/test/java/com/redhat/service/smartevents/integration/tests/steps). This folder contains also [hooks](src/test/java/com/redhat/service/smartevents/integration/tests/steps/Hooks.java) taking care of environment initialization and cleanup.
```

Comment:
A minor comment - These paths needs to be updated

Reply:
In v1 as well

Reply:
Thanks, updated.